content/specs/versions-macos-xcode-13-0.md

```diff
@@ -1,6 +1,6 @@
 ---
 description: A list of tools available out-of-the-box on Codemagic build machines.
-title: macOS build machine specification (Xcode 13.0 - 13.2)
+title: macOS build machine specification (Xcode 13.0 - 13.2 / Unity)
 aliases:
   - '../releases-and-versions/versions4'
   - '../specs/versions4'

```

Comment:
let's add versions-macos-xcode-13-0-unity as alias too

Reply:
addressed in https://github.com/codemagic-ci-cd/codemagic-docs/commit/1f1ef5af5afcf2180ba9e0d62afcb83dc0347a17